homeassistant/components/mqtt/vacuum/schema_legacy.py

```diff
@@ -226,7 +226,7 @@ def _setup_from_config(self, config):
                 CONF_FAN_SPEED_TOPIC,
             )
         }
-        self._templates = {
+        self._templates: dict[str, MqttValueTemplate] = {
```

Comment:
```suggestion
        self._templates: dict[str, MqttValueTemplate | None] = {
```

Reply:
The return type can be None here.

Reply:
I've removed the type hint for now. We can add in follow-up PR...